nle/env/base.py

```diff
@@ -201,7 +201,7 @@ def __init__(
 
         if self.savedir:
             self._ttyrec_pattern = os.path.join(
-                self.savedir, "nle.%i.%%i.ttyrec" % os.getpid()
+                self.savedir, "nle.%i.%%i.ttyrec.bz2" % os.getpid()
```

Comment:
pid is always going to be unique within the folder? Are we saving all ttyrecs to there or only the last N?

Reply:
Currently we are either saving all ttyrecs or none.

The pid is going to be the process id of the user's process which won't be reassigned until that process stops.